src/forms/previousExpenses.js

```diff
@@ -336,7 +277,8 @@ const PreviousExpensesStep = function ( props ) {
         clarifier = {'Monthly expenses that were expected to happen during the 12 months following that assessment'}
         left      = {{name: 'Previous', func: props.previousStep}}
         right     = {{name: 'Next', func: props.nextStep}}>
-        <ExpensesFormContent {...props} client={props.pageState} />
+        <ExpensesFormContent {...props} client={props.pageState}
+      />
```

Comment:
I think what you mean to do is move the  ' > ' char on line 279 dow on its own line and keep the '/>' on 281 on line 280.

Reply:
No, I meant to move them both. The `/` lets you know that the node is self-closing.